webview-ui/src/Kaito/KaitoModels.module.css

```diff
@@ -0,0 +1,163 @@
+body {
+    color: var(--vscode-editor-foreground, #fff);
+}
+
+.mainDiv {
+    margin-bottom: 2rem;
+}
+
+.modelsHeader {
+    font-size: 26px;
+    font-style: normal;
+    font-weight: 600;
+    line-height: normal;
+    margin: 1rem 0 0.5rem 0;
+}
+
+.subHeader {
+    width: 100vw;
+    max-width: 41rem;
+    margin-top: 0.9rem;
+    color: var(--vscode-editor-foreground, #fff);
+    opacity: 0.95;
+    font-size: 0.8125rem;
+    font-style: normal;
+    font-weight: 400;
+    line-height: normal;
+    word-wrap: break-word;
+    margin-bottom: 0.6rem;
+}
+
+.dropdown {
+    margin: 0;
+}
+
+.dropdownButton {
+    color: var(--vscode-editor-foreground, #fff);
+    font-size: 1.1rem;
+    font-style: normal;
+    font-weight: 700;
+    line-height: normal;
+    cursor: pointer;
+    border: none;
+    background: none;
+    display: flex;
+    align-items: center;
+    font-family: var(--vscode-font-family);
```

Comment:
I think do we change font families ? we can keep all fonts same?

Reply:
Oh yeah, when developing it, I realized that for whatever reason, the <button> element doesn't inherit the default vscode font, it uses Arial by default, so I just added this here to make sure it uses the same font as everything else.